test/unistd/symlink_on_nodefs.c

```diff
@@ -14,52 +15,39 @@
 #include <limits.h>
 #include <stdlib.h>
 
-int main() {
+void setup() {
   EM_ASM(
-    fs.mkdirSync('./new-directory', '0777');
-    fs.writeFileSync('./new-directory/test', 'Link it');
-    fs.symlinkSync(fs.realpathSync('./new-directory'), './symlink');
+    fs.mkdirSync('new-directory', '0777');
```

Comment:
what's the difference between `fs` and `FS`?

Reply:
`fs` is working directly with the real (node) filesystem.  (e.g. `import * as fs from "fs"` in node)

`FS` is the emscripten VFS.